### DISCUSSION TITLE
Commits tagged with discussion id

### DISCUSSION THREAD

**Discussion by sualeh** [community/community#3562] [Discussions]
When I tag a commit to a discussion id with a check-in comment like say "#426", it becomes a link in the commit, but there is no comment made in the discussion linking back to that commit. If I do the same with an issue, there is a link in the issue as well as the commit. Can discussions be made to work like issues in this sense?

**Comment by ocornut** [comment]
+1000
I sincerely believe Discussions are losing most of their appeal without that. Makes cross references impossible to track and reduce discoverability. This is one of the reason we stopped using Discussions.
